mws/data_mws_credentials.go

```diff
@@ -0,0 +1,30 @@
+package mws
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DataSourceMwsCredentials() *schema.Resource {
+	type mwsCredentialsData struct {
+		Ids map[string]string `json:"ids,omitempty" tf:"computed"`
+	}
+	return common.DataResource(mwsCredentialsData{}, func(ctx context.Context, e any, c *common.DatabricksClient) error {
+		data := e.(*mwsCredentialsData)
+		if c.AccountID == "" {
+			return fmt.Errorf("provider block is missing `account_id` property")
```

Comment:
This execution path isn't tested. Test is simple:

```
unc TestDataSourceMwsCredentials(t *testing.T) {
	qa.ResourceFixture{
		Fixtures: []qa.HTTPFixture{},
		Resource:    DataSourceMwsCredentials(),
		Read:        true,
		NonWritable: true,
		ID:          "_",
	}.ExpectError("provider block is missing `account_id` property")
}
```

Reply:
Unit tests updated as instructed. coverage 100%